flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ScalingMetricEvaluator.java

```diff
@@ -314,13 +315,19 @@ protected static Map<ScalingMetric, EvaluatedScalingMetric> evaluateGlobalMetric
         var out = new HashMap<ScalingMetric, EvaluatedScalingMetric>();
 
         var gcPressure = latest.getOrDefault(GC_PRESSURE, Double.NaN);
-        var lastHeapUsage = latest.getOrDefault(HEAP_USAGE, Double.NaN);
+        var lastHeapUsage = latest.getOrDefault(HEAP_MAX_USAGE_RATIO, Double.NaN);
 
         out.put(GC_PRESSURE, EvaluatedScalingMetric.of(gcPressure));
         out.put(
-                HEAP_USAGE,
+                HEAP_MAX_USAGE_RATIO,
                 new EvaluatedScalingMetric(
-                        lastHeapUsage, getAverageGlobalMetric(HEAP_USAGE, metricHistory)));
+                        lastHeapUsage,
+                        getAverageGlobalMetric(HEAP_MAX_USAGE_RATIO, metricHistory)));
+
+        var latestObservation = latest.getOrDefault(HEAP_AVERAGE_SIZE, Double.NaN);
+        double heapSizeAverage = getAverageGlobalMetric(HEAP_AVERAGE_SIZE, metricHistory);
+        LOG.debug("Latest: {}, Avg: {}", latestObservation, heapSizeAverage);
```

Comment:
Should we remove this?

Reply:
It's logged on debug, why not keep it?

Reply:
""Latest: 100, Avg: 400" doesn't really mean anything. We can keep it but then we should add some more info the log

Reply:
Removing.